### PR TITLE
libjwt: Added openssl to resolve broken CICD pipeline

### PR DESCRIPTION
**Pull request by bollig**

### CODE REVIEW

File: var/spack/repos/builtin/packages/libjwt/package.py [hunk 1]
```diff
@@ -21,4 +21,6 @@ class Libjwt(AutotoolsPackage):
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
     depends_on('m4', type='build')
+    # Needs openssl at runtime to ensure we can generate keys
+    depends_on('openssl')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/libjwt/package.py]: Is this only a `run` dependency or , maybe, `('build', 'run')`?

Comment (bollig) [reply var/spack/repos/builtin/packages/libjwt/package.py]: Yes. build, run.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/libjwt/package.py]: Never hurts to be explicit.  Thanks.

**Review by tldahlgren — APPROVED** [review]
Approval contingent on all required tests passing.

**Comment by tldahlgren** [comment]
Thank you for your contribution!

**Pull request merged**